internal/sdkprovider/provider/provider.go

```diff
@@ -36,10 +36,11 @@ func Provider(version string) *schema.Provider {
 		Schema: map[string]*schema.Schema{
 			"api_token": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
```

Comment:
because `DefaultFunc` exists, this becomes `true` in runtime, I have changed it for clarity